telemetry/specs/INT.mdk

```diff
@@ -366,9 +373,9 @@ cause the packet length to exceed egress link MTU, it
 **must not insert any metadata and set the M bit** in the
 INT header, indicating that egress MTU was exceeded at an INT hop.
 
-An INT source needs to be able to insert 8 bytes of INT header plus Per-hop
-Metadata Length\*4
-bytes of its own metadata. If inserting 8 bytes of INT header itself causes
+An INT source needs to be able to insert 12 bytes of fixed INT shim header and
```

Comment:
This number is 16 for INT over TCP/IP. 12 for VxLAN-GPE, 12 for Geneve.

Also, we need to clarify that this is "additional" INT overhead for which MTU needs to be tweaked, not accounting for MTU adjustments for VxLAN and Geneve itself.

Reply:
Another point to clarify is that this overhead assumes that only Per-hop INT metadata header is in use. No data is being piggybacked using Src-to-Sink INT metadata header.